### DISCUSSION TITLE
a11y warnings

### DISCUSSION THREAD

**Discussion by Schmell** [huntabyte/shadcn-svelte#1512] [Help] [answered]
I don't see anyone else complaining about this so maybe i have configured something wrong.
I get accesabilities warnings when using components.

``[vite-plugin-svelte] src/lib/components/ui/button/button.svelte:63:20  `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead``

i cant seem to fix this from the component file and it doesn't effect the component it is just annoying.

Thanks all

**Comment by shyakadavis** [comment]
@Schmell 

Are you using Svelte 4, or 5?

P.S. I repro would go a long way.

**Reply by Schmell** [reply]
```"svelte": "^5.0.0",```
To reproduce:
`npx sv create` - add most of the stuff (prettier, tailwind etc..)
follow [shadcn installation instructions](https://next.shadcn-svelte.com/docs/installation/sveltekit)
`npx shadcn-svelte@next init`
`npx shadcn-svelte@next add` - add everything cause why not
`npm run dev`

**Reply by MDr164** [reply]
Same here, this got triggered when I added paraglide.js from the `sv create` scaffolding in the beginning. Without it the warning is gone for me.

**Reply by Schmell** [reply]
Ahh. i installed paraglide as well.
Weird though. How can paraglide produce warnings taht come from the bits-ui/shadcn-svelte Librarys

**Reply by Bizordec** [reply, marked as answer]
Update your `@inlang/paraglide-sveltekit` package, this issue has been fixed in version `0.12.1`
https://github.com/opral/inlang-paraglide-js/issues/259
